### PR TITLE
Add multi-user project dashboard alert

### LINKED ISSUE #129: Single User Mode
We still need basic safeguards to ensure that two people cannot get on and edit the same project simultaneously - currently there is nothing in place to prevent that from happening, which, for obvious reasons, could lead to trouble.

### PR DESCRIPTION
**Pull request by akstuhl**
### What does this PR do?
- Adds a frontend alert from the data provided by the projects/dashboard API endpoint. If other users are recorded as editing a project, a new user upon opening that project will see an alert warning them of possible save conflicts and providing the names and email addresses of those other users.
- Fixes a very specific case in which auto project selection for guest users could trigger a loop
- Increases default rollover window size

### What issues does it address?
Closes #129 

### How to test
Log in to http://dm-staging.middell.net/ from a couple different accounts on different machines/browsers and open the same project. When opening a project for which an account is already signed in, you should see an alert dialog with red header text before you see the project's resource list. This dialog should contain names and email addresses for those other accounts and should have a "Continue" button to let you proceed to the resource list.

### CODE REVIEW

**Comment by SteveMarvin** [comment]
This is looking good, except for some violations being thrown. New issue written for those: **Violations on opening project #143.**

File: src/main/resources/static/js/dm/viewer/ProjectViewer.js [hunk 1]
```diff
@@ -409,6 +420,33 @@ dm.viewer.ProjectViewer.prototype.switchToProject = function(project, force) {
...
+            }
+            if (conflictingUsers.length > 0) {
+                returnState = this.state;
+                this.showModal(STATES.alert);
+                $("#alert-text").empty();
+                $("#alert-text").append("<p>Warning! Changes you make could be overwritten by other users. The following users are also logged in as editing this project:</p><ul id='alert-users'></ul>");
+                conflictingUsers.forEach(function(conflictingUser) {
+                  $("#alert-users").append("<li>" + conflictingUser.name + ": <a href='mailto:" + conflictingUser.email + "'>" + conflictingUser.email + "</a></li>");
+                });
+                $("#dismiss-alert").off("click").on("click", function() {
```
Comment (jamiefolsom) [inline src/main/resources/static/js/dm/viewer/ProjectViewer.js:445]: Hi @akstuhl -- mostly works great, but this doesn't appear to work -- clicking "Continue" button doesn't dismiss the dialog.

**Review by jamiefolsom — CHANGES_REQUESTED** [review]
Hey @akstuhl just one issue that I can see. Once that's fixed, ready to merge. Thanks!

File: src/main/resources/static/js/dm/viewer/ProjectViewer.js (hunk 1 above)
Comment (akstuhl) [reply src/main/resources/static/js/dm/viewer/ProjectViewer.js:445]: Thanks @jamiefolsom -- the Continue button should leave the modal open but proceed to the TOC. Steve found the button wasn't doing anything until he cleared his browser cache, then worked correctly.

Comment (jamiefolsom) [reply src/main/resources/static/js/dm/viewer/ProjectViewer.js:445]: ah, ok thx -- will re-check

**Review by jamiefolsom — APPROVED** [review]

File: src/main/resources/static/js/dm/viewer/ProjectViewer.js (hunk 1 above)
Comment (jamiefolsom) [reply src/main/resources/static/js/dm/viewer/ProjectViewer.js:445]: 👍

**Pull request merged**